lib/generators/komponent/styleguide_generator.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
```

Comment:
That file seems not belong to this PR.

Reply:
Hmmm looks like.

Reply:
Can you remove it ?

Reply:
Yes I will, sorry I was really busy those days

Reply:
Done